syntax_highlighting/fr/vscode/syntaxes/catalavs.xml

```diff
@@ -86,7 +86,7 @@
         </dict>
         <dict>
           <key>match</key>
-          <string>([^`#])</string>
+          <string>([^`])</string>
```

Comment:
Hum, why remove the `#` ? Shouldn't it be `<string>([^`#>])</string>` ?

Reply:
Little mc I think..